packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -58,7 +59,8 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
   constructor(
     public client: tsh.TshClient,
     private mainProcessClient: MainProcessClient,
-    private notificationsService: NotificationsService
+    private notificationsService: NotificationsService,
+    private usageReporterService: UsageEventService
```

Comment:
Following the previous comment on naming, it feels like this property should be renamed so that we use a consistent name across different files.

Reply:
Done